samples/pmic/native/npm1300_one_button/README.rst

```diff
@@ -0,0 +1,120 @@
+.. _npm1300_one_button:
+
+nPM1300: One button
+###################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The One button sample demonstrates how to support wake-up, shutdown and user interactions through
+a single button connected to the nPM1300.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+The sample also requires an nPM1300 EK.
+
+Overview
+********
+
+The sample uses a single button for controlling the device state.
+The sample controls an LED, load switch and the PMIC power mode based on button press duration.
+
+Wiring
+******
+
+#. Make the following connections on the nPM1300 EK:
+
+   * Remove all existing connections.
+   * Connect a suitable battery to the **J2** connector.
+   * On the **P17** pin header, connect **HOST** and **LED2** pins with a jumper.
+   * On the **P13** pin header, connect **RSET1** and **VSET1** pins with a jumper.
+   * On the **P14** pin header, connect **RSET2** and **VSET2** pins with a jumper.
+   * On the **LOAD SWITCH CONFIG** pin header, connect **VOUT2** and **LSIN2** pins with a jumper.
+
+#. Connect the chosen DK to the nPM1300 EK as in the following table:
+
+   .. list-table:: nPM1300 EK connections.
+      :widths: 25 25 25 25 25
+      :header-rows: 1
+
+      * - nPM1300 EK pins
+        - nRF52 DK pins
+        - nRF52840 DK pins
+        - nRF5340 DK pins
+        - nRF9160 DK pins
+      * - SDA
+        - P0.26
+        - P0.26
+        - P1.02
+        - P0.30
+      * - SCL
+        - P0.27
+        - P0.27
+        - P1.03
+        - P0.31
+      * - GPIO3
+        - P0.22
+        - P0.22
+        - P1.04
+        - P0.10
+      * - VDDIO
+        - VDD
+        - VDD
+        - VDD
+        - VDD
+      * - GND
+        - GND
+        - GND
+        - GND
+        - GND
+
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/pmic/native/npm1300_one_button`
+
+.. include:: /includes/build_and_run.txt
+
+Testing
+*******
+
+|test_sample|
+
+#. |connect_kit|
+#. |connect_terminal|
+
+If the initialization was successful, the terminal displays the following message with status information:
+
+.. code-block:: console
+
+   PMIC device ok
+
+
+Different length button presses
+
+.. _table::
+   :widths: auto
+
+   ============  ============  =====================================================
+   Duration      Log output    Outcome
+   ============  ============  =====================================================
+   < 1 sec       Short press   Sample flashes LED at 5Hz, LDSW 1 enabled
+   1 - 5 sec     Medium press  Sample flashes LED at 1Hz, LDSW 1 disabled
+   5 - 10 sec    Long press    Sample enters ship mode provided USB is not connected
+   > 10 sec      None          PMIC power cycles system
```

Comment:
There's no code for this? Is this hardware logic in PMIC?

Also, it is not clear from the user perspective what happens when "PMIC power cycles system", especially since host development kit is powered through USB.

Reply:
I'll clarify this.  Yes the 10 second time is HW reset by the PMIC.

Reply:
I think it would add more value to the sample if we mention this in description or in code comments.